tests/integration/cmd_url_test.go

```diff
@@ -121,14 +121,16 @@ var _ = Describe("odo url command tests", func() {
 			helper.CmdShouldPass("odo", "url", "create", "myurl", "--secure", "--context", context)
 
 			actualURLDescribeJSON := helper.CmdShouldPass("odo", "url", "describe", "myurl", "-o", "json", "--context", context)
-			desiredURLDescribeJSON := fmt.Sprintf(`{"kind":"url","apiVersion":"odo.dev/v1alpha1","metadata":{ "name": "myurl","creationTimestamp": null},"spec":{ "port": 8080,"secure": true},"status": {"state": "Not Pushed"}}`)
+			desiredURLDescribeJSON := fmt.Sprintf(`{"kind":"url","apiVersion":"odo.dev/v1alpha1","metadata":{ "name": "myurl","creationTimestamp": null},"spec":{"port": 8080,"secure": true,"kind": "route"},"status": {"state": "Not Pushed"}}`)
 			Expect(desiredURLDescribeJSON).Should(MatchJSON(actualURLDescribeJSON))
 
 			helper.CmdShouldPass("odo", "push", "--context", context)
 
 			// odo url describe -o json
 			actualURLDescribeJSON = helper.CmdShouldPass("odo", "url", "describe", "myurl", "-o", "json", "--context", context)
-			desiredURLDescribeJSON = fmt.Sprintf(`{"kind":"url","apiVersion":"odo.dev/v1alpha1","metadata":{ "name": "myurl","creationTimestamp": null},"spec":{ "port": 8080,"secure": true},"status": {"state": "Pushed"}}`)
+			fullURLPath := helper.DetermineRouteURL(context)
+			pathNoHTTP := strings.Split(fullURLPath, "//")[1]
```

Comment:
can you please add a comment regarding this?

Reply:
Sucre, comment added. 
It's to get the route URL, and actually this is exactly same code as 
https://github.com/openshift/odo/blob/3154f6010d9dc5f7dba66be0a81ddbd1c530023f/tests/integration/cmd_url_test.go#L152
and 
https://github.com/openshift/odo/blob/3154f6010d9dc5f7dba66be0a81ddbd1c530023f/tests/integration/cmd_url_test.go#L169-L170

Reply:
Thanks :+1: